### PR TITLE
v2v: Add automate methods for post-import network configuration

### PR DESCRIPTION
**Pull request by matobet**
Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1404920

Depends on ManageIQ/manageiq-automation_engine#35

### CODE REVIEW

**Comment by matobet** [comment]
@miq-bot assign @tinaafitz

**Comment by matobet** [comment]
@miq-bot add_label fine/yes

**Comment by matobet** [comment]
Depends on https://github.com/ManageIQ/manageiq-content/pull/123

File: content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/wait_for_vm_import.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ def main
                   status = vm.custom_get(:import_status)
                   case status
                   when 'success'
+                    @handle.set_state_var('imported_vm_id', vm.id)
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/wait_for_vm_import.rb]: @matobet, can we update the spec test?

File: content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/configure_vm_networks.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/configure_vm_networks.rb]: @matobet Can we add a test for this method?

File: content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/wait_for_vm_import.rb (hunk 1 above)
Comment (matobet) [inline content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/wait_for_vm_import.rb]: @tinaafitz done

File: content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/configure_vm_networks.rb (hunk 2 above)
Comment (matobet) [inline content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/configure_vm_networks.rb]: @tinaafitz done

**Comment by tinaafitz** [comment]
@gmcculloug Please review.

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 3863ee5d9bcbee8e13cfb49e8262cde2fd304509
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Jun 13 14:59:47 2017 -0400

    Merge pull request #123 from matobet/v2v-network-automate
    
    v2v: Add automate methods for post-import network configuration
    (cherry picked from commit 4167dfa8da3c38ec58db1eb1f2df05603da844a0)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1459996
```

**Pull request merged**